haystack/components/audio/whisper_local.py

```diff
@@ -111,19 +113,20 @@ def from_dict(cls, data: Dict[str, Any]) -> "LocalWhisperTranscriber":
     @component.output_types(documents=List[Document])
     def run(self, sources: List[Union[str, Path, ByteStream]], whisper_params: Optional[Dict[str, Any]] = None):
         """
-        Transcribes the audio files into a list of Documents, one for each input file.
+        Transcribes a list of audio files into a list of documents.
 
         For the supported audio formats, languages, and other parameters, see the
         [Whisper API documentation](https://platform.openai.com/docs/guides/speech-to-text) and the official Whisper
-        [github repo](https://github.com/openai/whisper).
+        [GitHup repo](https://github.com/openai/whisper).
 
         :param audio_files:
             A list of paths or binary streams to transcribe.
 
         :returns: A dictionary with the following keys:
-            - `documents`: A list of Documents, one for each file. The content of the document is the transcription
-                text, while the document's metadata contains the values returned by the Whisper model, such as the
-                alignment data and the path to the audio file used for the transcription.
+            - `documents`: A list of documents, one for each transcribed audio file. The content of the document
```

Comment:
```suggestion
            - `documents`: A list of documents where each document is a transcribed audio file. The content of the document
```
Otherwise it may sound like one list is for one transcribed file, I think